docs/topics/logging.txt

```diff
@@ -398,12 +408,43 @@ configure logging using your own approach), you can set
 :setting:`LOGGING_CONFIG` to ``None``. This will disable the
 configuration process.
 
-.. note::
-    Setting :setting:`LOGGING_CONFIG` to ``None`` only means that the
-    configuration process is disabled, not logging itself. If you
-    disable the configuration process, Django will still make logging
-    calls, falling back to whatever default logging behavior is
-    defined.
+Setting :setting:`LOGGING_CONFIG` to ``None`` only means that the configuration
+process is disabled, not logging itself. If you disable the configuration
+process, Django will still make logging calls, falling back to whatever default
+logging behavior is defined.
+
+Here's an example of how to make the logging system print everything to the
```

Comment:
The only example that should be included here is showing how to import logging and configure it yourself (if you want to bypass Django's default logging config); showing a sample alternate logging config is off-topic for this section.

We should also link to the docs for Django's default config here, so you know what configuration you're bypassing by using this approach.